src/main/kotlin/persistence/model/ILanguageEntity.kt

```diff
@@ -0,0 +1,24 @@
+package persistence.model
+
+import io.requery.*
+
+@Entity
+interface ILanguageEntity: Persistable {
+
+    @get:Key
+    @get:Generated
+    var id: Int
```

Comment:
if the id is autogenerated once a language is inserted into the database then we shouldn't be pulling the id of the language object from door43 right?

Reply:
Right, the door43 id shouldn't be mapped over to the Language object when being prepared for database insertion